### ISSUE TITLE
Please use new fork: machyve

### ISSUE THREAD

**Issue by yurikoles** [xhyve-xyz/xhyve#8]
https://github.com/machyve/machyve

**Comment by stv0g** [comment]
Hi Yuri,

why did you forked? It would be nice to know what you changed :-)

**Comment by yurikoles** [comment]
@stv0g I wanted to create new place to collaborate of all interesting parties. For now just renamed xhyve=>machyve.

**Comment by jeremyhu** [comment]
Yeah, I'm actually more interested in rolling things together than forking.  The whole reason I made xhyve-xyz is because @mist64 went missing for 6 months, and I decided to try and get some community motion around xhyve.  Since then, things kinda got rolling for a little bit, and I let it slide.

There has been a lot of motion in bhyve, and one of the things I want to do is rebase xhyve on more rencet bhyve, but that's not super trivial since @mist64 didn't document exactly what FBSD revision/branch it was forked from, and there are some general code churn differences that make it difficult to lock down empirically.

That being said, I'd really like to see bhyve and xhyve sharing a common codebase.

**Comment by stv0g** [comment]
I agree. Focusing community effort is good. But creating a new Repo / Fork 
/ Name does exactly the opposite.

There are too many versions of xhyve now.
Docker has done another fork under the name HyperKit.

Our efforts should be to merge existing forks and to motivate the existing 
owners of the repositories to give access to a broader community.

Steffen


Am 1. Juli 2017 1:26:07 vorm. schrieb Jeremy Huddleston Sequoia 
<notifications@github.com>:

**Comment by yurikoles** [comment]
@jeremyhu @stv0g I think new name is a must. Because we need to distinguish from other forks with same name. You had created a team, but please consider change of the project name and all mentions in code to `xhyve-xyz`.

**Comment by stv0g** [comment]
I am also not really happy with the xhyve-xyz name. I think we should 
convince mist64 to open his repo for more contributors and the close all 
the other forks.


Am 1. Juli 2017 8:59:56 vorm. schrieb Yurii Kolesnykov 
<notifications@github.com>:

**Comment by ondraknezour** [comment]
See https://github.com/mist64/xhyve/issues/124#issuecomment-356148474

**Comment by yurikoles** [comment]
@ondraknezour
Thanks for the link.

How to initiate poll on GitHub?
